Resources/Prototypes/Reagents/medicine.yml

```
@@ -164,13 +164,13 @@
         conditions:
         - !type:ReagentThreshold
           min: 30
-        probability: 0.02
+        probability: 0.02 
       - !type:Jitter
         conditions:
         - !type:ReagentThreshold
           min: 15
       - !type:Drunk
-  pricePerUnit: 4
+  pricePerUnit: 1 #3
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 2

(indentation)